### ISSUE TITLE
Overclock tool

### ISSUE THREAD

**Issue by FantasticMind** [steward-fu/gh_retrogame_emulator#70]
The settings don't stay on reboot.  It goes back to stock 528.

**Comment by jaZzKCS** [comment]
I do not know, but I am pretty sure this is intended behavior. It will revert on reboot anyways, but could be auto-set upon launch.

**Comment by FantasticMind** [comment]
Yeah or give the user the option to save the setting on reboot if they like.

**Comment by steward-fu** [comment]
save setting is not good idea because it might be boot loop forever.
I will try to add into next release for each emulator with overclock setting

**Comment by steward-fu** [comment]
each emulator can set itself overclock value in next release (v20180313)
thanks

**Comment by FantasticMind** [comment]
Great!  Thanks for your hard work!
